src/web/builds.rs

```diff
@@ -43,18 +43,22 @@ pub fn build_list_handler(req: &mut Request) -> IronResult<Response> {
     let mut conn = extension!(req, Pool).get()?;
     let limits = ctry!(req, Limits::for_crate(&mut conn, name));
 
+    let is_json = req.url.path().join("/").ends_with(".json");
```

Comment:
nit: this allocates on every request
```suggestion
    let is_json = req.url.path().last().map_or(false, |segment| segment.ends_with(".json"));
```
https://docs.rs/iron/0.6.1/iron/struct.Url.html#method.path
I do think this is a slight change in behavior in that `/builds.json/` will no longer work, but that seems like a misfeature anyway.